src/App.tsx

```diff
@@ -1,109 +1,73 @@
 import React from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import Pagination from './components/Pagination/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
-
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = React.useState<number>(5);
```

Comment:
If you pass a number, string, or boolean as initial value is is already have this type

Also you can use useState with out `React.` part and import it from 'react'